web/src/app/donate/DonateContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import DonateComponent from './DonateComponent';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { donateOperations } from './duck';
+
+class DonateContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  // TODO: make sure this user exists
```

Comment:
Would marking the prop as 'required' solve this TODO? -> https://reactjs.org/docs/typechecking-with-proptypes.html